rs/tests/driver/src/k8s/tnet.rs

```diff
@@ -409,25 +372,41 @@ impl TNet {
         )
         .await?;
 
-        if vm_type == ImageType::IcOsImage {
+        if vm_type == ImageType::IcOsImage || vm_type == ImageType::BoundaryImage {
             // create a job to download the image and extract it
-            let image_url = format!(
-                "http://server.bazel-remote.svc.cluster.local:8080/cas/{}",
-                match vm_req.primary_image {
-                    ImageLocation::IcOsImageViaUrl { url: _, sha256 } => sha256,
-                    _ => self.image_sha.clone(),
+            let image_url = match vm_type {
+                ImageType::IcOsImage => get_ic_os_img_url().expect("missing image url").to_string(),
+                _ => get_boundary_node_img_url()
+                    .expect("missing image url")
+                    .to_string(),
+            };
+            let config_image_url = format!(
+                "{}/{}/config_disk.img{}",
+                self.config_url.clone().unwrap(),
+                vm_name.clone(),
+                match vm_type {
+                    ImageType::IcOsImage => "",
+                    _ => ".zst",
                 }
-            );
+            )
+            .to_string();
             // TODO: only download it once and copy it if it's already downloaded
             let args = format!(
-                "set -e; \
+                "set -xe; \
                 mkdir -p /tnet/{vm_name}; \
-                wget -O /tnet/{vm_name}/img.tar.zst {image_url}; \
+                curl --retry 10 --retry-delay 1 -o /tnet/{vm_name}/img.tar.zst {image_url}; \
```

Comment:
Regarding this `curl` call and the ones below I would also set a `--user-agent` [like we do in Farm](https://github.com/dfinity-lab/farm/blob/master/backend/Farm/VMManager.hs#L1179) for logging purposes.